### DISCUSSION TITLE
How to detect if any value was changed in a dialog

### DISCUSSION THREAD

**Discussion by devalphabyte** [serenity-is/Serenity#5537] [Help] [answered]
I would like to know if I can detect if a field was modified in a dialog without binding an onchange event to each field

**Comment by VictorTomaili** [comment]
There is no default behavior for detecting changed values.

**Comment by devalphabyte** [comment, marked as answer]
I will be following this snippet for the resolution. https://github.com/serenity-is/Serene/blob/master/Serene/Serene.Core/Modules/Northwind/Customer/CustomerDialog.ts
